### PR TITLE
refactor(ivy): extract elementStart first create pass

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**

### CODE REVIEW

File: packages/core/src/render3/instructions/element.ts [hunk 1]
```diff
@@ -45,25 +67,26 @@ export function ɵɵelementStart(
...
+  ngDevMode && assertDataInRange(lView, adjustedIndex);
+
   const renderer = lView[RENDERER];
-  const native = lView[index + HEADER_OFFSET] = elementCreate(name, renderer, getNamespace());
-  const tNode = getOrCreateTNode(tView, lView[T_HOST], index, TNodeType.Element, name, attrs);
+  const native = lView[adjustedIndex] = elementCreate(name, renderer, getNamespace());
 
+  const tNode = tView.firstCreatePass ?
+      elementStartFirstCreatePass(index, tView, lView, native, name, attrsIndex, localRefsIndex) :
+      tView.data[index + HEADER_OFFSET] as TElementNode;
```
Comment (kara) [inline packages/core/src/render3/instructions/element.ts]:
```suggestion
      tView.data[adjustedIndex] as TElementNode;
```

**Review by kara — APPROVED** [review]
LGTM, one nit

File: packages/core/src/render3/instructions/element.ts (hunk 1 above)
Comment (pkozlowski-opensource) [reply packages/core/src/render3/instructions/element.ts]: Good catch, thnx!

**Comment by kara** [comment]
[VE presubmit](http://test/OCL:282013659:BASE:282013688:1574452593866:80a74c17)

[Ivy presubmit]()

**Pull request closed without merging**